packages/react-app-revamp/supabase_schemas/analytics_rewards_v3.sql

```diff
@@ -6,6 +6,7 @@ create table public.analytics_rewards_v3 (
     token_address character varying default null,
     amount_paid_in numeric default 0,   
     amount_paid_out numeric default 0,
-    created_at int4 null  
+    created_at int4 null,
+    amount_withdrawn numeric default 0 
```

Comment:
We should update this to align w the db which is `amount_withdrawn numeric null`.